examples/3_2_writing_data_GTFS.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Writing GTFS data\n",
+    "# 3.2. Writing data: GTFS\n",
```

Comment:
I found it weird that we skipped a section when numbering, I wonder if it's because we were a bit too ambitious and were planning to add functionality to write to a different model format soon https://github.com/arup-group/genet/issues/42

Reply:
Haha, yeah. Shall I renumber to remove this oddity?

Reply:
yeah i think what you did here with renumbering is good